openc3/lib/openc3/utilities/local_mode.rb

```diff
@@ -0,0 +1,507 @@
+# encoding: ascii-8bit
+
+# Copyright 2022 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+require 'fileutils'
+require 'json'
+# require 'openc3/models/gem_model' # These are used by also create circular dependency
+# require 'openc3/models/plugin_model' # These are used by also create circular dependency
+require 'openc3/utilities/s3'
+
+module OpenC3
+  module LocalMode
+    OPENC3_LOCAL_MODE_PATH = ENV['OPENC3_LOCAL_MODE_PATH'] || "/plugins"
```

Comment:
I don't see OPENC3_LOCAL_MODE_PATH being set anywhere else. Where should this point and should it be in the .env file?

Reply:
This is intended to be changed during unit testing because /plugins won't exist.